app/src/main/java/com/twilio/video/app/ui/room/RoomViewEvent.kt

```diff
@@ -7,7 +7,7 @@ sealed class RoomViewEvent {
     data class SelectAudioDevice(val device: AudioDevice) : RoomViewEvent()
     object ActivateAudioDevice : RoomViewEvent()
     object DeactivateAudioDevice : RoomViewEvent()
-    data class Connect(
```

Comment:
I'd prefer to keep the `Connect` verb here as CreateRoom can be confusing with the REST API operations available with Room.